esp/public/media/scripts/onsite/ajax_status.js

```diff
@@ -586,6 +588,10 @@ function render_table(display_mode, student_id)
             var section = data.sections[data.timeslots[ts_id].sections[i]];
             var parent_class = data.classes[section.class_id];
             
+            //  Hide walk-in seminars from display.
+            if (parent_class.category__id == open_class_category.id)
+                continue;
+            
```

Comment:
Following 30a888f, I added this to exclude the walk-in classes from the table as well as from the category options... which I think was the intent of #1465, but I'm open to correction. (There is an argument for keeping walk-in classes in the table so onsite volunteers can easily inform students of their choices; however, that would be an argument for showing walk-in activities among the category checkboxes also.)